controllers/storagecluster/storagecluster_controller.go

```diff
@@ -181,7 +182,8 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			},
 			enqueueStorageClusterRequest,
 		).
-		Watches(&ocsv1alpha1.StorageConsumer{}, enqueueStorageClusterRequest, builder.WithPredicates(ocsClientOperatorVersionPredicate))
+		Watches(&ocsv1alpha1.StorageConsumer{}, enqueueStorageClusterRequest, builder.WithPredicates(ocsClientOperatorVersionPredicate)).
+		Watches(&ocsclientv1a1.StorageClient{}, enqueueStorageClusterRequest)
```

Comment:
We should watch the created storageclient using Own and an owner reference

Reply:
done.